### DISCUSSION TITLE
Variable lists required for Land-Atmosphere Interactions comparison

### DISCUSSION THREAD

**Discussion by CharlesZheZhang** [NCAR/ESPAT_S2S#7] [General]
Provide a list of variables necessary for evaluating Land-Atmosphere Interactions for CESM and MPAS-NoahMP

**Comment by megandevlan** [comment]
The S2S reforecasts use this guidance for output fields: [S2S Namelist Public](https://docs.google.com/document/d/1NrOZNetP4AlpT0Cy2mL_lnSOEDukGyOpX_xm_EbDiGY/edit?usp=sharing). These include daily averages and 6-hourly instantaneous fields. It sounds like MPAS-NoahMP only outputs instantaneous fields, so we should think about how best to directly compare. 

I've also made a list based on Findell et al. (2024), which describes a data request to enable assessing land-atmosphere interactions across a range of temporal scales: 

**CAM** 
- (daily) cam.h1: TREFHT, QREFHT, PRECT, PRECC, PBLH, LHFLX, SHFLX, CLDTOT, CLDLIQ,
- (3-hourly) cam.h2: PRECT:A, PRECC:A, 'FLNT:A','FLNS:A, 'FSNT:A,'FSNS:A, 'FLDS:A,'FSDS:A, TREFHT:I, QREFHT:I, PS:I, PSL:I, PBLH:I, LHFLX:A, SHFLX:A
- (3-hourly) cam.h3: U:I, V:I, OMEGA:I, T:I, Q:I, PS:I

**CLM** 
- (daily) clm2.h1: FV:A, Rnet:A,  SOILWATER_10CM:A, TG:A, TSKIN:A
- (3-hourly) clm2.h2: SOILWATER_10CM:I, TSOI_10CM:I, QVEGE:A, QVEGT:A, QSOIL:A, SOILLIQ:A, Rnet:A

**Comment by CharlesZheZhang** [comment]
We put together a list of variables from MPAS-NoahMP outputs for diagnosing Land-Atmosphere Interactions:
[MPAS-NoahMP_Variables_for_L-A_Interactions](https://docs.google.com/document/d/1o_dOTsmnw8oYAV34zs-8ohmHwZd2IXGZjQiv6ArFIhc/edit?usp=sharing)
Many of them can be output every 3-hourly time interval and align with [Findell et al. (2024) paper](https://doi.org/10.5194/gmd-17-1869-2024). 

For MPAS, these output variables can be set in streams.atmosphere, stream_list.atmosphere.output, and stream_list.atmosphere.diagnostics.
